Cargo.toml

```diff
@@ -137,12 +137,21 @@ features = ["nightly", "push", "process"]
 git = "https://github.com/pingcap/rust-prometheus.git"
 rev = "b668f3911d6569de2e1e8b2672fccec1cc8298be"
 
-[replace]
+#[replace]
 # TODO: remove this when new raft-rs is published.
-"raft:0.6.0-alpha" = { git = "https://github.com/pingcap/raft-rs", branch = "master" }
-"raft-proto:0.6.0-alpha" = { git = "https://github.com/pingcap/raft-rs", branch = "master" }
-"protobuf:2.8.0" = { git = "https://github.com/nrc/rust-protobuf", branch = "v2.8" }
-"protobuf-codegen:2.8.0" = { git = "https://github.com/nrc/rust-protobuf", branch = "v2.8" }
+#"raft:0.6.0-alpha" = { git = "https://github.com/pingcap/raft-rs", branch = "master" }
```

Comment:
should we clean it？

Reply:
These comments are only temporary. I had to remove the `[replace]` section so I could use `[patch]` for the temporary git dependency replacement. I am going to revert both changes before merging, so this code will look the same as it did previously.